crates/bitwarden/src/vault/mod.rs

```diff
@@ -6,12 +6,7 @@ mod send;
 #[cfg(feature = "mobile")]
 mod totp;
 
-pub use cipher::{
-    attachment::{
-        Attachment, AttachmentEncryptResult, AttachmentFile, AttachmentFileView, AttachmentView,
-    },
-    Cipher, CipherListView, CipherView,
-};
+pub use cipher::*;
```

Comment:
So far we've shied away from glob imports, specially for public API, should we change this back?

Reply:
I'm split. It's a pain to re-export everything in every layer, the `cipher` is already a mod file.